near-contract-standards/src/fungible_token/storage_impl.rs

```diff
@@ -68,6 +68,15 @@ impl StorageManagement for FungibleToken {
         self.internal_storage_balance_of(&account_id).unwrap()
     }
 
+    /**
```

Comment:
Is it meant to be a documentation comment? In rust `///` is used for that and `//` for regular comments. I find `* *` difficult to read and I never observe this style in Rust source code, so I would like to ask you to chose either `///` or `//` for the comment blocks.

Reply:
My JS-trained hands did this automatically and my editor formatted it as a documentation comment and I figured I'd leave it in and see if anyone hollered 😄